PWGHF/Tasks/taskMultiplicityEstimatorCorrelation.cxx

```diff
@@ -0,0 +1,137 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file taskMultiplicityEstimatorCorrelation.cxx
+/// \brief Task for correlating the multiplicity estimator with generated dN/deta
+///
+/// \author Fabrizio Chinu <fabrizio.chinu@cern.ch>, Università and INFN Torino
+
+#include <string>
+#include <vector>
+#include <array>
+
+#include "TPDGCode.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/HistogramRegistry.h"
+#include "Framework/runDataProcessing.h"
+#include "Framework/StaticFor.h"
+#include "Common/DataModel/Multiplicity.h"
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+
+struct taskMultiplicityEstimatorCorrelation {
```

Comment:
```suggestion
struct HfTaskMultiplicityEstimatorCorrelation {
```

Reply:
Hi @vkucera, thanks, I didn't notice the Hf at the beginning of the tasks!

Reply:
Thanks